robot/markers.py

```diff
@@ -46,6 +46,45 @@ def rot_y_degrees(self) -> Degrees:
         return Degrees(math.degrees(self.rot_y_radians))
 
 
+_Orientation = NamedTuple('Orientation', (
+    ('rot_x_radians', Radians),
+    ('rot_y_radians', Radians),
+    ('rot_z_radians', Radians),
+))
+
+
+class Orientation(_Orientation):
+    """
+    Represents the orientation in 3d space as rotations around x, y, and z axes.
+
+    Rotations around the different axes can be thought of as follows:
+      - X rotation represents pitch. Positive rotation
+      around the X axis represents a rotation with the nearest point going downwards,
+      and furthest going upwards. (i.e. a bowing motion is a positive x rotation)
```

Comment:
It took me a couple of attempts to understand the second sentence here ("Positive rotation..."). I like that that approach applies quite easily to each of the descriptions, though it's a bit complex on first reading.

I wonder whether moving the example first would help people grok the idea sooner so that the technically-correct explanation then has an idea to hold onto?

Alternatively, is there a slightly different phrasing which is easier to grok first time?

(I'm assuming we'll use a similar phrasing in the main docs too, so I think this is worth getting right here)

Reply:
ca54c3e